.gitlab-ci.yml

```diff
@@ -736,6 +736,8 @@ library:ci-fiat_crypto_ocaml:
 
 library:ci-flocq:
   extends: .ci-template-flambda
+  script:
+    - ./configure COQEXTRAFLAGS="-compat 8.13"
```

Comment:
Not like this. You should edit `dev/ci/ci-flocq.sh`.

Reply:
That doesn't work, either.  If I try it locally, I see:

```
$ ./configure COQEXTRAFLAGS="-compat 8.13"
./configure.ml: Unknown option: COQEXTRAFLAGS=-compat 8.13.
```

Reply:
Are you testing with a very old version of Flocq (or at least `configure`)? Because it is certainly meant to work:

```console
$ ./configure COQEXTRAFLAGS="-compat 8.13"
...
$ grep compat Remakefile
	coqc -compat 8.13 -R src Flocq $<
```

Reply:
I rebases on master today, so I don't see how ./configure could be too old.

Reply:
Perhaps because `configure` is not part of the Git repository? Did you actually run `autoconf`?

Reply:
> Perhaps because `configure` is not part of the Git repository?

Don't think so.  configure exists, I ran it with the -local option after my test.

> Did you actually run `autoconf`?

No idea.  Check the job log. The problem is with configure.

Reply:
> I ran it with the -local option after my test.

Are you trying to pass `COQEXTRAFLAGS="-compat 8.13"` to the `configure` script of Coq? It is the `configure` script of Flocq that needs to receive this option. That is why Théo told you to modify `dev/ci/ci-flocq.sh`.

Reply:
I first tried it in `dev/ci/ci-flocq.sh` through CI, then confusedly tried to understand it better with Coq's `configure`.  I expect @SkySkimmer's comments below will get something that works.